scripts/translations/getTranslations.js

```diff
@@ -0,0 +1,101 @@
+/*
+  eslint
+  no-console: 0,
+*/
+const translate = require('@k3rn31p4nic/google-translate-api'); // @vitalets/google-translate-api
```

Comment:
I am a little weirded out by the scoped package. Why not https://www.npmjs.com/package/google-translate-api directly?

Reply:
That one isn't supported anymore and didn't work for me. There are a couple of supported forked versions and this was one of them and the only one that I could get to work consistently.